scripts/test_schemas.sh

```diff
@@ -6,7 +6,7 @@ if [ "$1" == "--local" ] || [ "$2" == "--local" ]; then
 fi
 
 if [ "$run_docker" == true ]; then
-    branch=v3
+    branch=add-routing-rules-to-primary-person-validator
```

Comment:
Please remember to revert this change before merging.

Reply:
Reverted